### PR TITLE
Add additional data into config file

### LINKED ISSUE #127: [Refactor] Abstractify colors
# Overview

Colors need to be changed for each component currently

# What Needs to Change

Refactor code to pull color data from the config file.

### PR DESCRIPTION
**Pull request by DedsecKnight**
Resolves #127 

Data added into config file: 
- Sponsors
- Team Member
- Keynote Speakers
- FAQ
- Footer Data

### CODE REVIEW

File: components/KeynoteSpeaker.tsx [hunk 1]
```diff
@@ -27,8 +27,14 @@ export default function KeynoteSpeaker(props) {
   return (
     <div className="flex w-[28rem] h-[9rem] md:mr-20 mr-16 my-4">
       <div
-        style={{ backgroundColor: props.cardColor.light, overflow: 'hidden' }}
-        className="w-1/4 rounded-l-md"
+        // style={{ backgroundColor: props.cardColor.light, overflow: 'hidden' }}
```
Comment (NickBurnett) [inline components/KeynoteSpeaker.tsx:30]: Can we remove this comment?

**Review by NickBurnett — DISMISSED** [review]
looks pretty good

**Comment by nam-t24** [comment]
I believe this takes care of the rest of the configurable class colors. We should probably review this pr and merge if it's good soon so that other branches can start utilizing the config colors

**Review by abdullah50053 — DISMISSED** [review]

**Review by MichaelZhao21 — CHANGES_REQUESTED** [review]
Hi! This is rly rly good; there's just a few small changes to make :33 

Something funny look at the notes I made reading through the code:
```
primary: #F2F3FF or bg-indigo-100 or #ECEEFF
primaryDark: #C1C8FF or #DCDEFF or #D0D5FF or indigo-500
secondary: #D8F8FF or aqua
secondaryDark: #B0F1FF or darkAqua
complementary: #FDECFF or purple-200
complementaryDark: #F8ACFF or red-300
```

**Review by NickBurnett — DISMISSED** [review]
Everything Michael said

File: components/PendingQuestion.tsx [hunk 2]
```diff
@@ -16,9 +16,10 @@ export default function PendingQuestion({ question }: PendingQuestionProps) {
   return (
     <div className="flex flex-row items-center gap-x-2 my-3">
       <svg height="14" width="14">
+        {/* !change bullet color */}
         <circle cx="7" cy="7" r="5" fill="#F8ACFF" />
```
[the remark this answers is not in the document]
Comment (MichaelZhao21) [reply components/PendingQuestion.tsx:20]: update tailwind is a bitch so u have to do this:
```html
      <svg height="14" width="14" className="fill-current text-complementaryDark">
        {/* !change bullet color */}
        <circle cx="7" cy="7" r="5" />
      </svg>
```

**Review by MichaelZhao21 — DISMISSED** [review]

**Review by MichaelZhao21 — APPROVED** [review]
lgtm!

**Review by MichaelZhao21 — CHANGES_REQUESTED** [review]
Ah so apparently there's a bunch of things in here that we _don't_ want in the config file??? (@abdullah50053 confirm)

**Comment by abdullah50053** [comment]
> Ah so apparently there's a bunch of things in here that we _don't_ want in the config file??? (@abdullah50053 confirm)
@MichaelZhao21:
That's correct, a lot of the things in here we're keeping in the DB. We need to keep the color and event config stuff and bin everything else

**Comment by MichaelZhao21** [comment]
I resolved all merge conflicts with develop, but I don't have the brain cells rn to remove everything else from the config file

**Comment by MichaelZhao21** [comment]
@nam-t24 Can we just recreate this PR with only @abdullah50053's wanted changes cause it's so much more painful to remove everything we don't want.

**Comment by nam-t24** [comment]
> @nam-t24 Can we just recreate this PR with only @abdullah50053's wanted changes cause it's so much more painful to remove everything we don't want.

yah that'll be good

**Comment by MichaelZhao21** [comment]
Irrelevent due to #216

**Pull request closed without merging**